contrib/gitian-descriptors/gitian-linux.yml

```diff
@@ -79,7 +85,11 @@ script: |
             echo "REAL=\`which -a ${i}-${prog}-8 | grep -v ${WRAP_DIR}/${i}-${prog} | head -1\`" >> ${WRAP_DIR}/${i}-${prog}
             echo 'export LD_PRELOAD=/usr/lib/x86_64-linux-gnu/faketime/libfaketime.so.1' >> ${WRAP_DIR}/${i}-${prog}
             echo "export FAKETIME=\"$1\"" >> ${WRAP_DIR}/${i}-${prog}
-            echo "exec \"\$REAL\" \"\$@\"" >> $WRAP_DIR/${i}-${prog}
+            if [ "${i:0:9}" = "powerpc64" ]; then
+                echo "exec \"\$REAL\" -mcpu=power8 -mtune=power9 \"\$@\"" >> $WRAP_DIR/${i}-${prog}
```

Comment:
This looks like something that should be handled in depends/hosts/linux.mk

Reply:
What if someone else wants to build for POWER7 (outside of gitian)?

Reply:
Is there a reason for specifying `-mcpu=power8` here at all?  VSX goes back to P7 and VMX goes back to P4…

Reply:
Maybe not. How far back can we safely go without a noticeable performance hit?

Would also be good to confirm #13203 still works if compiled for pre-POWER8.

Reply:
`vec_vsx_ld` is POWER7; but the SHA sigma instructions are exclusive to POWER8 and newer.  However, it looks like the merged code does check the hardware cap before using it, so it depends on compiler support, not build hardware support.  (That is, if my understanding is correct, one could build all the way down to -mcpu=generic and you'd still get speed on P8+.)